### PR TITLE
Upgrading Guide for Altis v20

### PR DESCRIPTION
**Pull request by mikelittle**
Added upgrading guide.
Changed examples to use version 20.
Addresses https://github.com/humanmade/product-dev/issues/1602

### CODE REVIEW

File: other-docs/guides/upgrading/v20.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+Altis now supports running Node.js applications alongside WordPress, utilizing WordPress as a headless API. See [our Node.js
+documentation](docs://cloud/nodejs/) for more information. This is an experimental feature, you can enable it in Local Server, and
+we welcome feedback.
+
+### WordPress 6.6.1
+
+WordPress 6.6 brings lots of new features, bug fixes, and improvements across the board. In particular, there are improvemnets
+in performance and accessibility, and new features and updates for the block editor (Gutenberg) and the block API.
+There have been theming improvements of interest to theme developers and designers
```
Comment (wisyhambolu) [inline other-docs/guides/upgrading/v20.md:65]: This line needs a full stop at the end of the sentence.

File: other-docs/guides/upgrading/v20.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+## Changes to the Altis platform
+
+Altis dev tools now come with wp-browser 4.x and Codeception 5.x This upgrade has some breaking changes. Please refer to
+the [wp-browser release notes](https://github.com/lucatume/wp-browser/releases/tag/4.0.0)
+and [Codeception release post ](https://codeception.com/07-28-2022/codeception-5.html)
+
+## PHP
+
+We have added experimental support for PHP 8.3. This is not yet recommended for production use but you can test your code with
+PHP 8.3 in local server
```
Comment (wisyhambolu) [inline other-docs/guides/upgrading/v20.md:49]: I think "local server" should be "Local Server" for consistency with the other mentions in this file.

File: other-docs/guides/upgrading/v20.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+both tools. See the [wp-browser release notes](https://github.com/lucatume/wp-browser/releases/tag/4.0.0)
+and [Codeception release post ](https://codeception.com/07-28-2022/codeception-5.html)
+
+Local server now supports TablePlus as well as Sequel Ace.
+
+We have incorporated many updates to modules and libraries in Altis to bring in important bug fixes and improvements.
+
+### Documentation
+
+Our developer focussed documentation has been improved again. As usual, feedback from our customers and partners is always welcome.
```
Comment (wisyhambolu) [inline other-docs/guides/upgrading/v20.md:86]: Interesting spelling fo "focussed" here, never knew it could be spelt with 'ss' instead of 's'. 🤔

**Review by wisyhambolu — APPROVED** [review]

File: other-docs/guides/upgrading/v20.md (hunk 3 above)
Comment (mikelittle) [reply other-docs/guides/upgrading/v20.md]: That's me mistyping and hitting on an alternate spelling! Fixed.

**Pull request merged**